db/migrate/20180802145655_add_address_to_schools.rb

```diff
@@ -0,0 +1,12 @@
+class AddAddressToSchools < ActiveRecord::Migration
```

Comment:
To clarify, we're adding address fields because in order to create a patron for the v2 route, it requires the teacher's school address. Am I understanding correctly?

Reply:
Yes @nnat425 you will pass the teacher's school's address to the microservice as the teacher's address.